Tests/HttpPipelineTests/__Snapshots__/HttpPipelineTests/testCookies.1.Conn.txt

```diff
@@ -1,8 +1,23 @@
+Step:
+-----
+ResponseEnded
+
+Request:
+--------
+URL: /
+Method: GET
+Headers: [
+  
+]
+Body: (Data, 0 bytes)
+
+Response:
+---------
 Status 200 (ok)
 Headers: [
-  Set-Cookie: user_id=123456
-  Set-Cookie: lang=es
   Content-Type: text/html; charset=utf8
+  Set-Cookie: lang=es
+  Set-Cookie: user_id=123456
```

Comment:
hm actually, are headers order dependent? what happens if you set the same cookie twice? does the last one override the first? 

if so,  maybe we shouldn't be sorting headers....